sixth.py

```diff
@@ -0,0 +1,16 @@
+
+string = "PQRQRQRQRQRQA" 
+iter(string)
```

Comment:
Would you please tell mw why you used iter(string) here.

Reply:
i was just checking for iteration. just ignore it